src/main/java/com/company/Tweet.java

```diff
@@ -17,15 +20,15 @@ public static void main(String[] args) {
         TwitterFactory tf = new TwitterFactory(cb.build());
         Twitter twitter = tf.getInstance();
         try{
-            if(args[0].length()>280){
+            if(args == null || args.length()==0 || args.length()>280){
```

Comment:
what happens in the scenario where the following string is passed in?  " "

Reply:
addressed. I did not realize you could not tweet just spaces!